package.json

```diff
@@ -8,43 +8,40 @@
     "url": "https://github.com/OfficeDev/office-ui-fabric-react"
   },
   "scripts": {
-    "_invalidateCache": "node ./scripts/invalidate-just-cache.js",
-    "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
-    "_rushBuild": "node common/scripts/install-run-rush.js build",
-    "_rushBuildToFabricResources": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
-    "_rushRebuild": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild --production --verbose --lint",
-    "_rushRebuildCi": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild --verbose --lint",
-    "_rushRebuildFast": "npm run _invalidateCache && node common/scripts/install-run-rush.js rebuild",
-    "postinstall": "npm run _rushInstall && npm run _rushBuildToFabricResources -- --npm-install-mode",
-    "test": "npm run _rushBuild",
+    "preinstall": "node ./scripts/use-yarn-please.js",
+    "postinstall": "node ./scripts/postinstall.js",
+    "buildto": "node ./scripts/monorepo/buildTo.js",
+    "builddemo": "node ./scripts/monorepo/buildTo.js fabric-website-resources --min",
     "vrtest": "cd apps && cd vr-tests && npm run screener",
     "a11ytest": "cd apps && cd a11y-tests && npm run test",
     "start": "cd apps && cd fabric-website-resources && npm start",
     "start-exp": "cd packages && cd experiments && npm start",
-    "rebuild": "npm run _rushRebuildFast",
-    "build": "npm run _rushRebuild",
-    "buildci": "npm run _rushRebuildCi",
-    "buildfast": "npm run _rushBuild",
-    "code-style": "node common/scripts/install-run-rush.js code-style",
-    "change": "node ./scripts/node_modules/beachball/bin/beachball.js change",
-    "rush-update": "node common/scripts/install-run-rush.js update",
-    "generate": "npm run rush-update",
```

Comment:
Since there's not an explicit update/generate command now, does that just mean the lockfile will get updated automatically every time someone installs? Or just every time someone adds a new dep?

Reply:
yarn.lock file only updates if you change the package.json